### PR TITLE
fix: keep customData when converting to ExcalidrawElement (#7654)

### PR DESCRIPTION
**Pull request by yubinTW**
for #7654 

if `customData` of an `ExcalidrawElementSkeleton` is provided, when using `convertToExcalidrawElements()` to convert `ExcalidrawElementSkeleton` to `ExcalidrawElement`,  it will not disappear.

### CODE REVIEW

File: packages/excalidraw/data/transform.test.ts [hunk 1]
```diff
@@ -309,6 +309,112 @@ describe("Test Transform", () => {
...
+        y: 400,
+        fileId: "dummy-file-id" as FileId,
+      },
+    ];
+    const convertedElements = convertToExcalidrawElements(
+      rawData as ExcalidrawElementSkeleton[],
+      opts,
+    );
+    convertedElements.forEach((ele) => {
+      expect(ele.customData).not.toBeDefined();
```
Comment (ad1992) [inline packages/excalidraw/data/transform.test.ts:414]: separate test not needed, its already recorded in snapshot

File: packages/excalidraw/data/transform.test.ts [hunk 2]
```diff
@@ -309,6 +309,112 @@ describe("Test Transform", () => {
...
+      },
+      {
+        type: "text",
+        x: 100,
+        y: 400,
+        customData: { createdBy: "user01" },
+        text: "Hello",
+      },
+      {
+        type: "image",
```
Comment (ad1992) [inline packages/excalidraw/data/transform.test.ts:352]: just a single element is enough, also lets move this test towards the end

**Review by ad1992 — CHANGES_REQUESTED** [review]
Thanks for the PR @yubinTW

**Comment by yubinTW** [comment]
Hi @ad1992,
Thanks for the reviews, I had updated the test case.

**Comment by krzysztofMlczk** [comment]
@ad1992 can this be merged?

**Review by ad1992 — CHANGES_REQUESTED** [review]
@yubinTW can you also update the package changelog ? Also this should under Fixes category in changelog

**Comment by yubinTW** [comment]
Hi @ad1992
Sure, updated the CHANGELOG.

**Comment by krzysztofMlczk** [comment]
@ad1992 when can we expect patch with those changes?

**Review by ad1992 — APPROVED** [review]

**Comment by ad1992** [comment]
> @ad1992 when can we expect patch with those changes?

very soon, I will check if there is anything else we should push to patch and release by tonight/tomorrow.

**Comment by ad1992** [comment]
> @ad1992 how can I know that the patch is out?

I will be sharing here once the patch is released so post that you can update it

**Pull request merged**